packages/pages-components/src/map/providers/mapQuest.ts

```diff
@@ -60,6 +68,7 @@ function load(resolve, reject, apiKey, { version = "v1.3.2" } = {}) {
   const mapScript = document.createElement("script");
   mapScript.src = baseUrl + ".js";
   mapScript.onload = () => {
+    //@ts-ignore
```

Comment:
Is there a type mismatch here which is why you needed to ignore?

Reply:
removed